### PR TITLE
Fix mingw64 build

### PR DESCRIPTION
**Pull request by ypsvlq**
- [x] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

### CODE REVIEW

File: libr/include/heap/r_windows.h [hunk 1]
```diff
@@ -993,25 +993,25 @@ typedef struct _HeapInformation {
...
 
-NTSTATUS (NTAPI *RtlQueryProcessDebugInformation)(
+static NTSTATUS (NTAPI *RtlQueryProcessDebugInformation)(
 	IN DWORD ProcessId,
 	IN DWORD DebugInfoClassMask,
 	IN OUT PDEBUG_BUFFER DebugBuffer
 );
 
-NTSTATUS (NTAPI *RtlDestroyQueryDebugBuffer)(
+static NTSTATUS (NTAPI *RtlDestroyQueryDebugBuffer)(
```
[the remark this answers is not in the document]
Comment (trufae) [reply libr/include/heap/r_windows.h:1007]: i think all those function signatures should specify they use the WINAPI calling convention

File: libr/core/meson.build [hunk 2]
```diff
@@ -72,9 +72,6 @@ r_core_sources = [
 ]
 
 r_core_inc = []
-if host_machine.system() == 'windows'
```
Comment (trufae) [inline libr/core/meson.build:75]: 👍

**Review by trufae — CHANGES_REQUESTED** [review]

File: libr/include/heap/r_windows.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (ypsvlq) [reply libr/include/heap/r_windows.h:1007]: The function pointers have NTAPI, which seems correct per winternl.h

[the remark this answers is not in the document]
Comment (trufae) [reply libr/include/heap/r_windows.h:1007]: oh im blind :D 👍

**Pull request merged**